kernel/enter.c

```diff
@@ -820,6 +939,104 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     if (__myst_kernel_args.perf)
         _print_boottime();
 
+    // open the appenv.json
+    char* data;
+    size_t size;
+    char* path = "appenv.json";
+    if (myst_load_file(path, (void**)&data, &size) != 0)
+    {
+        goto skip_docker_env;
```

Comment:
Instead of adding a goto flow, can you put these code (942 - 1038) in a function? That is, the function can do early return if myst_load_file() != 0

Reply:
Implemented this as a different function, thank you for the suggestion